src/connectors/naxosmusiclibrary.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const playerBar = '#music';
+
+function getCatalogID() {
+	return Util.getTextFromSelectors(`${playerBar} .u-album`);
+}
+
+Connector.useMediaSessionApi();
+
+Connector.playerSelector = '.u-control.u-process';
+
+Connector.artistSelector = `${playerBar} .u-artist`;
+
+Connector.trackSelector = `${playerBar} .u-music-title`;
+
+Connector.getAlbum = () => {
+	const catalogID = getCatalogID();
+	const albumHeader = Util.queryElements(`.eps._playAlbum[data-catalogueid="${catalogID}"]`);
+	return albumHeader[0].textContent;
```

Comment:
Should we assert the array length and return null on zero length to avoid this blowing up if nothing is matched?

Reply:
Good catch. Force-pushed a revision.